cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -250,6 +250,26 @@ function testWorkflow(registry: string, repo: string, name: string) {
       cy.contains('button', 'Restub').click();
       cy.contains('button', 'Publish').should('be.disabled');
     });
+    it('hide and un-hide a version', () => {
```

Comment:
It would be great to create a function for these hiding and un-hiding as the workflow test and the tool test are near identical. However, when I tried moving this into a function I ran into problems.

Reply:
They seem quite a bit different, unfortunately; going to leave the same for now.